docs/wiki/framework/cargo-framework.md

```diff
@@ -114,7 +114,17 @@ Note that this function can be used to make objects loadable/unloadable (set to
  * [object, 3] call ace_cargo_fnc_setSize
 ```
 
-### 4.3 Adjusting cargo space of a vehicle
+### 4.3 Setting Custom Load/Unload Time
+
+```sqf
+object setVariable ["ace_cargo_delay", <TIME IN SECONDS>]
+```
+
+This will override the load/unload time calculation and instead set it to the value as specified. This value will _not_ be affected by any scaling coefficients, but will instead always take as long as the set delay.
+
+To disable, use `object setVariable ["ace_cargo_delay", -1]`.
```

Comment:
Why not just set to `nil`?

Reply:
either is supported, really. won't have an effect either way

Reply:
Yeah I just figured removing the variable might make more sense to people

Reply:
Mentioned both methods to fully document behaviour